tempodb/backend/raw.go

```diff
@@ -25,7 +26,15 @@ const (
 // from the backend
 type KeyPath []string
 
-type Feature int
+// FundFunc is executed for each object in the backend.  The provided FindOpts
+// are used to determine how to handle the object.  Any collection of these
+// objects is the callers responsibility.
+type FindFunc func(FindOpts)
+
+type FindOpts struct {
```

Comment:
I'd like to rename this to `FindResult` or something similar, to disambiguate it from the options pattern.

Reply:
How about `FindMatch` or something to indicate we're matching these sets of fields against the objects.

Reply:
I've pushed a commit for this change.  Let me know if that name seems better.